google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -119,7 +120,13 @@ private Subscriber(Builder builder) throws IOException {
             Ints.saturatedCast(ackExpirationPadding.getStandardSeconds()));
     clock = builder.clock.isPresent() ? builder.clock.get() : CurrentMillisClock.getDefaultClock();
 
-    flowController = new FlowController(builder.flowControlSettings);
+    flowController =
+        new FlowController(
+            builder
+                .flowControlSettings
+                .toBuilder()
+                .setLimitExceededBehavior(LimitExceededBehavior.ThrowException)
```

Comment:
Michael is right, we can't allow the use of BLOCK or IGNORE.

So let me know what do you think is best, override the user setting or throw an IllegalArgumentException? I honestly think that overriding the value is fine, since message-receiving-actions are not initiated by the user so the limitexceededbehavior is basically meaningless to the user on the subscriber side.